api/pkg/transformer/pipeline/testdata/valid_feast_expression.yaml

```diff
@@ -0,0 +1,79 @@
+transformerConfig:
+  preprocess:
+    inputs:
+      - variables:
+          - name: customer_id
+            jsonPath: $.customer.id
+          - name: customer_level
+            literal:
+              intValue: 10
+      - tables:
+          - name: driver_table
+            baseTable:
+              fromJson:
+                jsonPath: $.drivers[*]
+                addRowNumber: true
+      - feast:
+          - tableName: driver_feature_table
+            project: default
+            entities:
+              - name: driver_id
+                valueType: STRING
+                jsonPath: $.drivers[*].id
+            features:
+              - name: driver_feature_1
+                valueType: INT64
+                defaultValue: '0'
+              - name: driver_feature_2
+                valueType: INT64
+                defaultValue: '0'
+          - tableName: customer_feature_table
+            project: default
+            entities:
+              - name: customer_id
+                valueType: STRING
+                expression: customer_id
+              - name: customer_level
+                valueType: INT32
+                expression: customer_level
```

Comment:
Can you also add UDF expression here? Such as Now() or Geohash()

Reply:
Added